packages/eslint-config-eslint/default.yml

```diff
@@ -2,93 +2,93 @@ extends:
     "eslint:recommended"
 
 rules:
-    array-callback-return: 2
-    indent: [2, 4, {SwitchCase: 1}]
-    block-spacing: 2
-    brace-style: [2, "1tbs"]
-    camelcase: [2, { properties: "never" }]
-    callback-return: [2, ["cb", "callback", "next"]]
-    comma-spacing: 2
-    comma-style: [2, "last"]
-    consistent-return: 2
-    curly: [2, "all"]
-    default-case: 2
-    dot-notation: [2, { allowKeywords: true }]
-    eol-last: 2
-    eqeqeq: 2
-    func-style: [2, "declaration"]
-    guard-for-in: 2
-    key-spacing: [2, { beforeColon: false, afterColon: true }]
-    keyword-spacing: 2
-    new-cap: 2
-    new-parens: 2
-    no-alert: 2
-    no-array-constructor: 2
-    no-caller: 2
+    array-callback-return: "error"
+    indent: ["error", 4, {SwitchCase: 1}]
+    block-spacing: "error"
+    brace-style: ["error", "1tbs"]
+    camelcase: ["error", { properties: "never" }]
```

Comment:
That really looks nice, especially in the `.yml` files which I think are meant to be fairly readable. This is great.